translation/init-crowdin-project.sh

```diff
@@ -9,38 +9,44 @@
 
 
 # Stable versions
-PLF_VERSION=6.3.x
-CHAT_ADDON_VERSION=3.3.x
-NEWS_VERSION=2.3.x
-TASK_ADDON_VERSION=3.3.x
-WEB_CONFERENCING_ADDON_VERSION=2.3.x
-PUSH_NOTIFICATIONS_ADDON_VERSION=2.3.x
-WALLET_ADDON_VERSION=2.3.x
-KUDOS_ADDON_VERSION=2.3.x
-PERKSTORE_ADDON_VERSION=2.3.x
-GAMIFICATION_VERSION=2.3.x
-ONLYOFFICE_VERSION=2.3.x
-APPCENTER_VERSION=2.3.x
-DIGITAL_WORKPLACE_VERSION=1.3.x
-MEEDS_VERSION=1.3.x
-AGENDA_VERSION=1.2.x
-JITSI_VERSION=1.2.x
-ANALYTICS_VERSION=1.2.x
-NOTES_VERSION=1.1.x
-POLL_VERSION=1.0.x
-GAMIFICATION_GITHUB_VERSION=1.0.x
-MAIL_INTEGRATION_VERSION=1.0.x
-MULTIFACTOR_AUTHENTICATION_VERSION=1.1.x
-AUTOMATIC_TRANSLATION_VERSION=1.0.x
-DOCUMENTS_VERSION=1.0.x
-PROCESSES_VERSION=1.0.x
-AGENDA_CONNECTORS_VERSION=1.0.x
+PLF_VERSION=6.4.x
+CHAT_ADDON_VERSION=3.4.x
+NEWS_VERSION=2.4.x
+TASK_ADDON_VERSION=3.4.x
+WEB_CONFERENCING_ADDON_VERSION=2.4.x
+PUSH_NOTIFICATIONS_ADDON_VERSION=2.4.x
+WALLET_ADDON_VERSION=2.4.x-exo
+KUDOS_ADDON_VERSION=2.4.x-exo
+PERKSTORE_ADDON_VERSION=2.4.x-exo
+GAMIFICATION_VERSION=2.4.x-exo
+ONLYOFFICE_VERSION=2.4.x
+APPCENTER_VERSION=2.4.x
+DIGITAL_WORKPLACE_VERSION=1.4.x
+MEEDS_VERSION=1.4.x-exo
+AGENDA_VERSION=1.3.x
+JITSI_VERSION=1.3.x
+ANALYTICS_VERSION=1.3.x
+NOTES_VERSION=1.2.x
+POLL_VERSION=1.1.x
+GAMIFICATION_GITHUB_VERSION=1.1.x-exo
+MAIL_INTEGRATION_VERSION=1.1.x
+MULTIFACTOR_AUTHENTICATION_VERSION=1.2.x
+AUTOMATIC_TRANSLATION_VERSION=1.1.x
+DOCUMENTS_VERSION=1.1.x
+PROCESSES_VERSION=1.1.x
+AGENDA_CONNECTORS_VERSION=1.1.x
+DLP_VERSION=1.0.x
+ANTI_BRUTEFORCE_VERSION=1.0.x
+ANTI_MALWARE_VERSION=1.0.x
+DEEDS_DAPP_VERSION=1.0.x
+DEEDS_TENANT_VERSION=1.0.x
```

Comment:
We don't need them, they continue using latest crowdin project even we don't have stable branches for them